plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraTransform.kt

```diff
@@ -62,4 +63,25 @@ internal object CameraTransform {
   }
 
   fun calculateScaleBy(amount: Double, currentZoom: Double) = log2(amount) + currentZoom
+
+  /**
+   * Extension function to wrap longitude to one world copy.
+   */
+  fun Point.wrapCoordinate(): Point {
+    val lng = wrap(this.longitude(), -LONGITUDE_MAX.toDouble(), LONGITUDE_MAX.toDouble())
+    return Point.fromLngLat(lng, this.latitude())
+  }
+
+  // Wrap value to the given range (including min, excluding max).
+  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+  internal fun wrap(value: Double, min: Double, max: Double): Double {
+    if (value >= min && value < max) {
+      return value
+    } else if (value == max) {
```

Comment:
We should not use equals for Double equality testing. Maybe we need to convert to `BigDecimal` or some other solution.

Reply:
good point. i will update that to use some precision. that will be better than `BigDecimal` i think.